### ISSUE TITLE
LittleFX [BUG] Effects Triggering when near the block, even if you dont touch it

### ISSUE THREAD

**Issue by itsMarkiS** [Rsl1122/Little-Things#10]
I Do use the plugin with some blocks and cactus as well, but i noticed that when you walk near the cactus you get the effect triggered, even if you are not touching it which is a weird behaviour 

it happens not only with cactus, but with every block, you get the effect by walking one block away from it, and idk if it is intended, but it feels like it is not supposed to happen, and imo it should be like the honey block

"like the honey block??" yes, the honey block, when you stick/touch on it you get a "slow" effect (not a potion effect)
as you can see in the video. But when you walk near them without touching it, you dont get this slow

https://user-images.githubusercontent.com/47507632/221004822-9d036885-ce39-4d9a-b9b6-ac14c017bed4.mp4

**Comment by itsMarkiS** [comment]
Uploading Minecraft_ 1.19.2 - Multiplayer (3rd-party Server) 2023-02-23 18-13-58.mp4…

**Comment by AuroraLS3** [comment]
Yes this is how it is intended to work since it's easier to check what player is next to than checking if player is touching a block. I'll see what I can do though
